docs/static/env-vars.asciidoc

```diff
@@ -1,20 +1,20 @@
 [[environment-variables]]
-=== Using environment variables in the configuration
+=== Using environment variables
 
 ==== Overview
 
 * You can set environment variable references in the configuration for Logstash plugins by using `${var}`.
-* At Logstash startup, each reference will be replaced by the value of the environment variable.
+* At Logstash startup, each reference is replaced by the value of the environment variable.
 * The replacement is case-sensitive.
 * References to undefined variables raise a Logstash configuration error.
 * You can give a default value by using the form `${var:default value}`. Logstash uses the default value if the
 environment variable is undefined.
-* You can add environment variable references in any plugin option type : string, number, boolean, array, or hash.
+* You can add environment variable references in any plugin option type: string, number, boolean, array, or hash.
```

Comment:
```suggestion
* You can add environment variable references in any <<plugin-value-types,plugin option type>>: string, number, boolean, array, or hash.
```
Perhaps a dumb question: Are the "plugin option types" are the same as these described [value types](https://www.elastic.co/guide/en/logstash/8.2/configuration-file-structure.html#plugin-value-types)? If so, we may want to add a link.